lib/trailblazer/activity/circuit.rb

```diff
@@ -76,7 +76,7 @@ def to_h
       private
 
       def next_for(last_task, signal)
-        outputs = @map[last_task]
+        outputs = @map.fetch(last_task, {})
```

Comment:
Unfortunately, we cannot simply return an empty hash when there literally is no next step. We already found a workaround which is included in the next release.

Reply:
Well IMHO you're not returning an empty hash but a signal (or nil) :

method definition :

```ruby
def next_for(last_task, signal)
  outputs = @map.fetch(last_task, {})
  outputs[signal]
end
```

The `fetch` is here to prevent `NoMethodError`  :

```sh
NoMethodError: undefined method `[]' for nil (NoMethodError)

        outputs[signal]
```